content/plugin/sdkv2/testing/acceptance-tests/index.mdx

```diff
@@ -83,45 +62,187 @@ signature:
+
+Each provider may have additional requirements and setup recommendations. Refer to the provider's codebase for more details.
+
+### Terraform CLI Installation Behaviors
+
+The testing framework implements the following Terraform CLI discovery and installation behaviors:
+
+- If the `TF_ACC_TERRAFORM_PATH` environment variable is set, the framework will use that Terraform CLI binary if it exists and is executable. If the framework cannot find the binary or it is not executable, the framework returns an error unless the `TF_ACC_TERRAFORM_VERSION` environment variable is also set.
+- If the `TF_ACC_TERRAFORM_VERSION` environment variable is set, the framework will install and use that Terraform CLI version.
+- If both the `TF_ACC_TERRAFORM_PATH` and `TF_ACC_TERRAFORM_VERSION` environment variables are unset, the framework will search for the Terraform CLI binary based on the operating system `PATH`. If the framework cannot find the specified binary, it installs the latest available Terraform CLI binary.
+
+Refer to the [Environment Variables](#environment-variables) section for more details about behaviors and valid configurations.
+
 ## Running Acceptance Tests
 
-A `terraform` binary of version 0.12.26 or above must be available on the local system for the acceptance test driver to run, either on the system `$PATH` or supplied via the `TF_ACC_TERRAFORM_PATH` environment variable described below. If you are running acceptance tests in a CI environment, we recommend that you download and install Terraform with a prior build step.
+Ensure that the [acceptance testing requirements](#requirements-and-recommendations) are met and then use the [`go test`](https://pkg.go.dev/cmd/go/internal/test) command to run acceptance tests. You can run the acceptance tests on any environment capable of running `go test`, such as a local workstation [command line](#command-line), or continuous integration runner, such as [GitHub Actions](#github-actions).
 
-Terraform requires an environment variable `TF_ACC` be set in order to run
-acceptance tests. This is by design, and intended to prevent developers from
-incurring unintended charges when running tests. The easiest way to run
-acceptance tests is to use the built in `make` step `testacc`, which explicitly
-sets the `TF_ACC=true` value for you. Example:
+~> **Note**: Acceptance tests typically create and destroy actual infrastructure resources, possibly incurring expenses during or after the test duration.
 
+### Running Tests with the Command Line
+
+Run acceptance testing via the command line of any workstation. Use these instructions as the basis for other environments such as continuous integration runners.
+
+The following example will typically execute all available acceptance tests in a provider codebase:
```

Comment:
When we say "typically" it makes me feel like we need to explain why. Can we take it out?

Reply:
Providers can technically implement non-`go test` based testing. I'll remove `typically` for the 99% case though.

```suggestion
The following example will execute all available acceptance tests in a provider codebase:
```